array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,12 @@
+/* exported compact */
+function compact(array) {
+  var compactedArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (!!array[i] === false) {
```

Comment:
You could shorten this to use one if statement. 
`if (array[i]) {
  // run code to push array[i] into compactedArray
 }`
 
 This is accomplished because all the forbidden elements are [falsey](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values.